src/interpreter.rs

```diff
@@ -797,8 +806,10 @@ impl<'a, 'tcx: 'a, 'arena> Interpreter<'a, 'tcx, 'arena> {
         match self.mir().lvalue_ty(self.tcx, lvalue) {
             LvalueTy::Ty { ty } => self.type_repr(ty),
             LvalueTy::Downcast { adt_def, substs, variant_index } => {
-                let field_tys = adt_def.variants[variant_index].fields.iter()
-                    .map(|f| f.ty(self.tcx, substs));
+                let field_tys = adt_def.variants[variant_index]
+                                    .fields
+                                    .iter()
+                                    .map(|f| f.ty(self.tcx, substs));
```

Comment:
I don't want to use alignment. Next-line method calls should just be 4-space indented past the previous line.